modules/stanford_soe_helper_collection_layouts/stanford_soe_helper_collection_layouts.info

```diff
@@ -10,19 +10,28 @@ dependencies[] = ctools
 dependencies[] = features
 dependencies[] = image
 dependencies[] = pathauto
+dependencies[] = stanford_magazine
+dependencies[] = stanford_news_extras
+dependencies[] = stanford_people_spotlight
```

Comment:
@minorwm, Are the dependencies on stanford_news_extras or stanford_people_spotlight needed?